### DISCUSSION TITLE
테스트 접속 링크 사용법 관련 질문

### DISCUSSION THREAD

**Discussion by devinhyeok** [naverz/zepeto-studio-kor#248] [5. Q&A]
![image](https://user-images.githubusercontent.com/94621970/150731425-6204636a-4561-4816-8177-3788cfe9997c.png)
https://studio.zepeto.me/guides/testing-your-world

"[파일 등록] 탭에서 [테스트 접속 링크 복사하기] 버튼을 누르면, ZEPETO에서 테스트 대상 월드에 접속할 수 있는 링크를 얻을 수 있습니다.
해당 링크를 테스터들에게 공유해서 자유롭게 테스트를 진행해보세요!"
라고 되어있는데

이 링크로 제페토 앱에서 테스트 해볼려면 어떻게 해야하나요?

아무리 찾아도 보이질 않아요

**Comment by lhj5709** [comment]
DeepLink 앱을 설치하시고 거기에 입력하면 가능합니다

**Comment by hyeonjin-jo** [comment]
안녕하세요.

다양한 방법이 존재하는데 아래의 링크로 QR code를 만들어서 사용하는 방법도 좋을 것 같습니다.

https://ko.qr-code-generator.com/?ut_source=google_c&ut_medium=cpc&ut_campaign=&ut_content=&ut_term=_&gclid=CjwKCAiAlrSPBhBaEiwAuLSDUDJQgqPt9SuJDSdoPZKGurin2NchIDankbd-QVBYzJk2y2A5Wgo_2xoClEEQAvD_BwE

현재 관련 문의가 종종 들어오고 있어서 내부적으로 개선점을 찾고 있습니다.

참고 부탁드립니다.

감사합니다.
